src/host/HostSharePCI.cc

```diff
@@ -387,21 +555,28 @@ int HostSharePCI::set_pci_address(VectorAttribute * pci_device,
         return -1;
     }
 
-    oss << showbase << internal << setfill('0') << hex << setw(4) << ibus;
-
-    pci_device->replace("VM_BUS", oss.str());
-
     // --------------------- SLOT (PCI_ID +1) -----------------------
-    oss.str("");
-
     pci_device->vector_value("PCI_ID", slot);
 
     slot = slot + 1;
 
+    if ( bus_index )
+    {
+        ibus = slot;
+        slot = 0;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We observed that the presence of non-empty statements like `a + b` without clear context or purpose within the codebase can lead to potential confusion or misinterpretation. To mitigate this issue, it&#39;s advisable to ensure that such statements are appropriately integrated into larger constructs that contribute meaningfully to the program&#39;s logic or functionality. This could involve assigning the result of the expression to a variable.